src/components/popUpToDoListConfirmation.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    class="modal fade fixed top-0 z-10 left-0 w-full h-full outline-none overflow-x-hidden overflow-y-auto"
```

Comment:
This template is the same as the `ConfirmDelete.vue` Please do not duplicate the code.